### PR TITLE
doc: Add design doc for thin-lv clones

### PR DESCRIPTION
**Pull request by Yuggupta27**
Add design document to allow the creation of PVC-PVC clones
for PVCs backed by thinly-provisioned logical volumes.

Signed-off-by: Yug Gupta <yuggupta27@gmail.com>

### CODE REVIEW

**Review by cupnes — CHANGES_REQUESTED** [review]
Please make some corrections.

**Comment by Yuggupta27** [comment]
Thanks for the review @cupnes! Updated accordingly.
Also, attempted to address review comments from https://github.com/topolvm/topolvm/pull/446 here too.
Please let me know if I missed anything. Thanks!

**Review by cupnes — APPROVED** [review]

**Review by satoru-takeuchi — COMMENTED** [review]
I don't have an additional comment than [this](https://github.com/topolvm/topolvm/pull/447#discussion_r923203567). Please reflect this.

**Review by satoru-takeuchi — APPROVED** [review]

File: docs/proposals/thin-lv-clone.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    $ lvcreate -s --name thinsnap VG0/thinvolume
+    ```
+
+* STEP 2: Activation of the logical volume
+
+  A thinly-provisioned snapshot will be activated as Read-Write for Clone scenarios; so that it can be provided as a Persistent Volume object to kubernetes.
+
+    For e.g: Activating a logical volume for Read-Write operations:
+    ```bash
+    $ lvchange -ay -K VG0/thinsnap
```
[the remark this answers is not in the document]
Comment (Yuggupta27) [reply docs/proposals/thin-lv-clone.md]: Thanks! Updated :+1:

**Review by satoru-takeuchi — APPROVED** [review]

**Review by cupnes — APPROVED** [review]

**Pull request merged**